lib/pgBackRest/Db.pm

```diff
@@ -1022,16 +1037,18 @@ sub dbObjectGet
             }
         }
 
-        # Make sure the standby database is defined when backup from standby requested
+        # Make sure a standby database is defined when backup from standby option is set
         if (cfgOption(CFGOPT_BACKUP_STANDBY) && !defined($oDbStandby))
         {
-            confess &log(ERROR, 'unable to find standby database - cannot proceed');
+            # Throw an error that is distinct from connecting to the master for testing purposes
+            confess &log(ERROR, 'unable to find standby database - cannot proceed', ERROR_HOST_CONNECT);
         }
```

Comment:
New tests in RealAllTest make sure each of these separate code blocks are hit. Would a new error code of ERROR_DB_STANDBY_CONNECT be better? Or maybe I can create a DB Unit Test module to test the dbObjectGet function given the error messages are different, but that would still mean in the Real Tests we can't be sure which code block was hit.